src/nyc_trees/sass/partials/_survey.scss

```diff
@@ -254,8 +316,8 @@
     background: #f3f3f3;
     margin-bottom: -10px;
     position: absolute;
-    left: 0;
-    right: 0;
+    left: -2rem;
+    right: -2rem;
```

Comment:
Changing these 2 lines introduces a slight regression: it causes a horizontal scroll on the treecorder for mobile.